tests/features/environment.feature

```diff
@@ -11,11 +11,11 @@
    When  user calls selectEnv method by providing the name of one of these environments
    Then  the provided environment is selected
 
-  Scenario: Call selectEnv with the name of an unexisting environment
+  Scenario: Call selectEnv with the name of a non existing environment
```

Comment:
We should avoid general statements like this :
` several registered environments`
`user receives all available environments`
Instead, specify `environment A and B and registered` and `user receives environments A and B`